postgres-appliance/scripts/configure_spilo.py

```diff
@@ -417,7 +418,12 @@ def get_placeholders(provider):
             else:
                 placeholders['CALLBACK_SCRIPT'] = 'patroni_aws'
 
-    placeholders['USE_WALE'] = bool(placeholders.get('WAL_S3_BUCKET') or placeholders.get('WAL_GCS_BUCKET') or placeholders.get('WAL_SWIFT_BUCKET'))
+    use_walg = str(placeholders['USE_WALG']).lower() == 'true' and bool(placeholders.get('WAL_S3_BUCKET'))
+    placeholders['USE_WALE'] = bool(use_walg or placeholders.get('WAL_GCS_BUCKET')
```

Comment:
does it mean that from now on we can stream to S3 using only `wal-g` ?  Also, when I try 
```bash
 docker run -it -e WAL_S3_BUCKET="test-bucket"  spilo-test-walg:latest
```

the WALE_ENV_DIR is not written